examples/google_adk/README.md

```diff
@@ -0,0 +1,120 @@
+# ADK Example
+
+This example demonstrates training a Google ADK (Application Development Kit) agent using Agent-Lightning with reinforcement learning. The agent processes user instructions and generates ADK actions for Google applications. It's compatible with Agent-lightning v0.2 or later.
+
+## Requirements
```

Comment:
If you have decided to create a how-to tutorial, this README can be greatly simplified and only keeps necessarily installation and running guidance.